build.gradle

```diff
@@ -128,12 +128,5 @@ private void configureAndroidLibrary(Project project) {
             sourceCompatibility JavaVersion.VERSION_1_8
             targetCompatibility JavaVersion.VERSION_1_8
         }
-
-        buildTypes {
-            release {
-                minifyEnabled true
-                proguardFiles 'proguard-rules.pro'
-            }
-        }
```

Comment:
will this fix the issue where our private/protected items become public?

Reply:
Since Proguard won’t be run, field visibility won’t be modified, true. I think this change will also fix our stack traces and “step-in” capability when debugging from a consuming application.